schema/user.schema.graphql

```diff
@@ -68,11 +68,6 @@ type User @entity {
   bitcoinTransfers: [BitcoinTransfer!] @derivedFrom(field: "user")
   fastBTCBridgeStats: [FastBTCBridgeStat!] @derivedFrom(field: "user")
   """
-  Cross ChainBridge transfers from RSK and to RSK
```

Comment:
what is this doing here?

Reply:
I was trying to get this to deploy, and it was throwing errors about null properties for non-null values with sender and receiver. Don't re-review this PR yet - I want to see if I can get it to deploy first